src/Projection/ProjectionValidator.php

```diff
@@ -0,0 +1,18 @@
+<?php declare(strict_types=1);
+
+namespace Comquer\Projection;
+
+use Comquer\Validation\ArrayValidator;
+use Comquer\Validation\ArrayMissingRequiredKeysException;
+
+class ProjectionValidator
+{
+    public static function validateSerialized(string $projectionName, array $requiredKeys, array $serializedProjection): void
+    {
+        try {
+            ArrayValidator::validateMultipleKeysExist($requiredKeys, $serializedProjection);
+        } catch (ArrayMissingRequiredKeysException $exception) {
+            throw ProjectionValidatorException::missingRequiredKeysInSerializedProjection($projectionName, $exception->getMissingKeys());
+        }
+    }
+}
```

Comment:
@jakubgiminski Please fix line endings here and across entire PR. Line should end with line feed otherwise its not matching POSIX standard and can cause problems like this [one](http://gcc.gnu.org/ml/gcc/2003-11/msg01568.html)